terraform/aws/README.md

```diff
@@ -20,19 +20,33 @@ And input Access Key ID and Secret Access Key.
 
 ### Fill variables
 Copy example variables file `terraform.example.tfvars` to `terraform.tfvars` in the `terraform/aws` directory.
+Most variables are optional and terraform will prompt you for variables if you have not filled required variables.
 You can also override variables defined in `variables.tf`
 
 ### Run terraform apply
 Go to the `terraform/aws` directory and run following command in terminal:
 ```
+# you will need to run terraform init if terraform has yet to be initialiized
+# a .terraform.lock.hcl file will be present if terraform is initialized
+terraform init 
+
 terraform apply
 ```
 
 After calculating the diff you will be asked if you want to apply changes. Type `yes` and press `enter`.
 
 ### Get kubectl config for created cluster
+If you are using AWS CLI version ealier than 1.16.156 you will need to install `aws-iam-authenticator` before you can run `aws eks`. 
```

Comment:
It seems that differences in command sequence is caused by different versions of AWS CLI.
This instruction is mostly about AWS CLI v2.

Your changes are about AWS CLI v1. And they could be very helpful for people using this version. So I would like to ask you to move your suggestions to the separate note paragraph with title something like `Additional steps for AWS CLI v1`. Don't you think that it would allow us to better structure the doc, do you?

Reply:
https://github.com/zestymarket/graph-deployment/pull/1